Makefile

```diff
@@ -0,0 +1,47 @@
+BOLD := \033[1m
+RESET := \033[0m
+
+# -- Docker
+COMPOSE              = docker-compose
+COMPOSE_RUN          = $(COMPOSE) run --rm
+COMPOSE_RUN_APP      = $(COMPOSE_RUN) app
+
+# ==============================================================================
+# RULES
+
+default: help
+
+bootstrap: ## Bootstrap nextcloud project
+bootstrap: \
+	build \
+	install
+.PHONY: bootstrap
+
+build: ## Build the app container
+	@$(COMPOSE) build nextcloud;
+.PHONY: build
+
+install: ## Install Nextcloud
+	@$(COMPOSE) up -d db
+	@echo "Wait for postgresql to be up..."
+	@$(COMPOSE_RUN) dockerize -wait tcp://db:5432 -timeout 60s
+	@$(COMPOSE_RUN) nextcloud-install;
+.phony: install
+
+run: ## start the development server using Docker
+	@$(COMPOSE) up -d nextcloud
+	@echo "Wait for postgresql to be up..."
+	@$(COMPOSE_RUN) dockerize -wait tcp://db:5432 -timeout 60s
+.PHONY: run
+
+stop: ## stop the development server using Docker
+	@$(COMPOSE) stop
+.PHONY: stop
+
+# -- Misc
+
+help:  ## Show this help
+	@echo "$(BOLD)Nextcloud Makefile$(RESET)"
+	@echo "Please use 'make $(BOLD)target$(RESET)' where $(BOLD)target$(RESET) is one of:"
+	@grep -h ':\s\+##' Makefile | column -tn -s# | awk -F ":" '{ print "  $(BOLD)" $$1 "$(RESET)" $$2 }'
+.PHONY: help
```

Comment:
Please use the [simpler one as in Richie](https://github.com/openfun/richie/blob/master/Makefile#L298) (we also should migrate marsha to this).